packages/compiler/core/projector.ts

```diff
@@ -255,6 +256,10 @@ export function createProjector(
   }
 
   function projectModel(model: Model): Type {
+    if (model.name === "Foo") {
```

Comment:
```suggestion
    if (model.name === "Foo") {
```

Left over from debugging?

Reply:
indeed